nautobot/dcim/tests/test_api.py

```diff
@@ -1490,6 +1490,58 @@ def setUpTestData(cls):
         ]
 
 
+class InterfaceTestVersion13(InterfaceTestVersion12):
+    api_version = "1.3"
+
+    @classmethod
+    def setUpTestData(cls):
+        manufacturer = Manufacturer.objects.create(name="Test Manufacturer 1", slug="test-manufacturer-1")
+        devicetype = DeviceType.objects.create(manufacturer=manufacturer, model="Device Type 1", slug="device-type-1")
+        site = Site.objects.create(name="Site 1", slug="site-1")
+        devicerole = DeviceRole.objects.create(name="Test Device Role 1", slug="test-device-role-1", color="ff0000")
+        device = Device.objects.create(device_type=devicetype, device_role=devicerole, name="Device 1", site=site)
+
+        Interface.objects.create(device=device, name="Interface 1", type="1000base-t")
+        Interface.objects.create(device=device, name="Interface 2", type="1000base-t")
+        Interface.objects.create(device=device, name="Interface 3", type="1000base-t")
```

Comment:
Would probably make sense to explicitly set `status` on these created Interfaces in this test case at least.